addons/link_tracker/views/link_tracker_views.xml

```diff
@@ -44,7 +44,7 @@
                                 <field name="short_url"/>
                             </group>
                             <group name="utm" string="UTM">
-                                <field name="campaign_id"/>
+                                <field name="campaign_id" options="{'create_name_field': 'title', 'always_reload': True}"/>
```

Comment:
can't we overwrite the name_create instead ?

Reply:
Overwriting `name_create` would solve the "quick creation" issue, but it would not solve the other issue where the name entered in the many2one widget is not copied in the creation form when clicking on "Create and edit".
This is the purpose of adding the `create_name_field` option in the different views.

But I agree that having to add this everywhere where it is used is annoying and probably not the best option.
I checked a little bit and we could achieve the same result by also making the title a computed (and stored) field:

```py
@api.depends('name')
def _compute_title(self):
    for campaign in self:
        if self.name and not self.title:
            self.title = self.name
```
This way, when clicking on "Create and edit", the title will be computed using the name entered in the many2one widget and shown in the creation form.
But I think it's maybe odd to have the field name that depends on the field title, and the field title that depends on the field name.

Do you have a preference between the two solutions I proposed ? Or maybe do you have a better idea ?
Thanks in advance :)

Reply:
> But I think it's maybe odd to have the field name that depends on the field title, and the field title that depends on the field name.

Yes you are right :/ you can keep the current solution so